uwblueprint/project-read-backend
# add swagger ui api docs

### Notion ticket link



[API Documentation](https://www.notion.so/uwblueprintexecs/Set-up-API-documentation-72b6538023594347b58b190e688c4ef8)



### Implementation description

- Generated open-api schema
- Installed rest framework swagger UI
- Routed api schema to swagger HTML template



### Steps to test

1. Make sure you can log into Django Admin
2. go to /docs
3. play around



### What should reviewers focus on?

- if it works
- if all the endpoints show up

### Checklist
